.docker/prod.Dockerfile

```diff
@@ -1,4 +1,5 @@
 ARG VERSION=unstable
```

Comment:
Should be edge now right?
```suggestion
ARG VERSION=edge
```

Reply:
The build didn't work with edge but I can try changing it in another PR after this one is merged.